src/cityreader/cityreader.py

```diff
@@ -14,20 +13,36 @@
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
+import csv
+
+class City():
+  def __init__(self, name, lat, lon):
+    self.name = name
+    self.lat = lat
+    self.lon = lon
+
 cities = []
 
 def cityreader(cities=[]):
   # TODO Implement the functionality to read from the 'cities.csv' file
   # For each city record, create a new City instance and add it to the 
   # `cities` list
     
-    return cities
+  # NOTE use open and reader object 
+  with open('cities.csv', 'r') as csvCities: #r read files
+      reader = csv.reader(csvCities, delimiter = ',', 
+  quotechar = '|') # split a string with multiple delimiters (commas) acts as a field delimiter in a sequence of comma-separated values
+      for row in reader:
+        if row[0] != 'city':
+          cities.append(City(row[0], float(row[3]), 
```

Comment:
Great work with the conditional here to skip the first line. Alternatively, you can use `next(reader)` to skip the header.